
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      <link rel="icon" href="../../img/idec-wiki-logo.png">
      <meta name="generator" content="mkdocs-1.4.0, mkdocs-material-8.5.6">
    
    
      
        <title>Results - iDEC 2022 | Edinburgh-UHAS_Ghana</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.20d9efc8.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.cbb835fc.min.css">
        
          
          
          <meta name="theme-color" content="#02a6f2">
        
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="light-blue" data-md-color-accent="blue">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#results" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="iDEC 2022 | Edinburgh-UHAS_Ghana" class="md-header__button md-logo" aria-label="iDEC 2022 | Edinburgh-UHAS_Ghana" data-md-component="logo">
      
  <img src="../../img/idec-wiki-logo-white-bg.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            iDEC 2022 | Edinburgh-UHAS_Ghana
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Results
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/idec-teams/2022_Edinburgh-UHAS_Ghana" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="iDEC 2022 | Edinburgh-UHAS_Ghana" class="md-nav__button md-logo" aria-label="iDEC 2022 | Edinburgh-UHAS_Ghana" data-md-component="logo">
      
  <img src="../../img/idec-wiki-logo-white-bg.png" alt="logo">

    </a>
    iDEC 2022 | Edinburgh-UHAS_Ghana
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/idec-teams/2022_Edinburgh-UHAS_Ghana" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        Home
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../team/" class="md-nav__link">
        Team
      </a>
    </li>
  

    
      
      
      

  
  
    
  
  
    
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_3" type="checkbox" id="__nav_3" checked>
      
      
      
      
        <label class="md-nav__link" for="__nav_3">
          Project
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Project" data-md-level="1">
        <label class="md-nav__title" for="__nav_3">
          <span class="md-nav__icon md-icon"></span>
          Project
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../background/" class="md-nav__link">
        Background
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../design/" class="md-nav__link">
        Design
      </a>
    </li>
  

            
          
            
              
  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          Results
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        Results
      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#results-summary" class="md-nav__link">
    Results Summary
  </a>
  
    <nav class="md-nav" aria-label="Results Summary">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#wet-lab" class="md-nav__link">
    Wet Lab
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#modelling-and-docking" class="md-nav__link">
    Modelling and Docking
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#results-and-discussion-section-of-the-report" class="md-nav__link">
    Results and Discussion section of the report:
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../report/" class="md-nav__link">
        <b>Report</b>
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../supplementary_info/" class="md-nav__link">
        <b>Supplementary Information</b>
      </a>
    </li>
  

    
      
      
      

  
  
  
    
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_6" type="checkbox" id="__nav_6" >
      
      
      
      
        <label class="md-nav__link" for="__nav_6">
          Documentation
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Documentation" data-md-level="1">
        <label class="md-nav__title" for="__nav_6">
          <span class="md-nav__icon md-icon"></span>
          Documentation
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../documentation/protocols/" class="md-nav__link">
        Protocols
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../documentation/notebook/" class="md-nav__link">
        Lab notebook
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../documentation/data/" class="md-nav__link">
        Data
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#results-summary" class="md-nav__link">
    Results Summary
  </a>
  
    <nav class="md-nav" aria-label="Results Summary">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#wet-lab" class="md-nav__link">
    Wet Lab
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#modelling-and-docking" class="md-nav__link">
    Modelling and Docking
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#results-and-discussion-section-of-the-report" class="md-nav__link">
    Results and Discussion section of the report:
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  <a href="https://github.com/idec-teams/2022_Edinburgh-UHAS_Ghana/blob/main/docs/project/results.md" title="Edit this page" class="md-content__button md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>


<h1 id="results">Results</h1>
<h3 id="results-summary">Results Summary</h3>
<hr>

<h5 id="wet-lab">Wet Lab</h5>
<p>The (Maximum Inhibitory Concentrations) MICs of our untransformed strains were tested for, in low AgNO<sub>3</sub> concentrations. It was observed that even at concentrations as low as, 8 mg/L TOP10 cells did not grow. Thus, they were not considered the best cell strain for developing the selection. A sharp growth fall-off in BL21(DE3) cells was observed after 16 mg/L of AgNO<sub>3</sub>. This allowed us to explore BL21(DE3) cells as our expression strain for developing the selection method. </p>
<p>As described in the design section, a random mutant colony and its wild-type variant for <em>D. rerio, M. edulis, M. galloprovincialis,</em> and <em>S. cerevisiae</em> were put through the silver-toxicity selection method. We found that MT expressing BL21(DE3) cells had a higher MIC than the untransformed cells, suggesting that MT expression increases silver resistance in <em>E. coli</em>. There were no significant differences in the MIC of the wild-types and the mutants (growth fall-off at 20 mg/L of AgNO<sub>3</sub> was observed for all cells), suggesting that a screen for these mutants would be necessary to find any improved metallothionein mutants. </p>
<h5 id="modelling-and-docking">Modelling and Docking</h5>
<p>Modelling and Docking simulations generated results for the number of Ag<sup>+</sup> ions docked to each species of MT, the bind energies for each cysteine of each tested metallothionein was also generated (<a href="../../supplementary_info/">see Supplementary Information</a>). This allowed us to create a pass/fail metric based on whether the ion docking reaction would be spontaneous. Using this, total Gibbs free energy for each metallothionein was calculated along with calculating, energy of binding per ion. This gave us an in-silico analysis for which MT had the highest overall binding affinity towards Ag+ ions along with each MT’s binding “efficiency.” We found, that <em>P. fluorescens</em> which has the fewest number of cysteines, binds the greatest number of ions, therefore having the lowest total Gibbs free energy and Gibbs free energy of binding per ion (-0.407 kcal/mol). This suggests that <em>P. fluorescens</em> has a high binding efficiency and affinity for Ag<sup>+</sup> ions. Contrastingly, MTs like <em>M. edulis</em> contain more cysteines but bind to fewer Ag<sup>+</sup> ions thus having a lower binding efficiency and affinity. </p>
<h3 id="results-and-discussion-section-of-the-report">Results and Discussion section of the report:</h3>
<p><i><b>Determining inhibitory concentration of BL21(DE3)</b></i></p>
<p>To establish the MIC of untransformed BL21(DE3) and TOP10 cells, they were plated on a range of AgNO<sub>3</sub> concentrations starting with 8 mg/L until a fall-off in growth was observed (16 mg/L for BL21(DE3) cells). Cells were observed forming lawns until 16 mg/L and 17 mg/L with a sharp decline after that. This allowed us to explore 16 mg/L as the starting baseline concentration for the MT expressing cells. Contrastingly, TOP10 cells did not grow on any selected concentration suggesting that their silver resistance was too low to setup a functional selection pressure (Figure 1).   </p>
<figure>
<img src="https://github.com/idec-teams/2022_Edinburgh-UHAS_Ghana/blob/main/figures/results-1.png?raw=true" style="width: 50vw; left: 0; right: 0; display: block; margin: auto;">
<figcaption><b>Figure 1.</b> The cultured strains were plated in duplicate on AgNO<sub>3</sub> plates from 8-18 mg/L. The number of colonies grew on the plate was counted, with a solid colour representing a bacterial lawn and blank indicating no growth. It was observed that there is a growth fall-off at 16 mg/L for BL21(DE3) cells while the lowest concentrations of AgNO<sub>3</sub> was observed to be toxic towards TOP10 cells. </figcaption>
</figure>

<p><i><b>Error prone PCR mutant library generation</b></i></p>
<p>Error prone (EP) PCR was used to generate a mutant library of MTs (<a href="../design/">see Design section</a>). A higher concentration of dTTPs was used which would increase the chance of mutations with Uracil in the transcript. This is favourable for UGU and UGC as outcomes, which are the codons for cysteine (Section 4.3).  </p>
<p>The mutated PCR products were double-stranded linear fragments of DNA which were assembled into the ORF of pJUMP29-1A(lacZ) level 1 plasmids containing a constitutive promoter (J23100), using level 0 parts (<a href="../../supplementary_info/">see Supplementary Information S.1</a>) following the JUMP assembly protocol (Valenzuela-Ortega and French, 2021). The same assembly method was used to generate WT MT expression plasmids.  </p>
<p>All Level 1 plasmids were transformed into TOP10 cells and subjected to blue-white colony screening where blue colonies represent lacZ positive control. White colonies were checked for containing the insert using colony PCR (Section 4.2.2). A single WT MT expressing colony was picked and cultured to be transformed into BL21(DE3) cells for protein expression and negative selection. Similarly, a random mutant colony was picked, verified for containing the insert and cultured for BL21(DE3) transformation and subsequent selection. The mutants and wild type MT expressing cells were plated on AgNO<sub>3</sub> plates of increasing concentration (16-30 mg/L).  </p>
<h4 style="text-align: center;">Negative selection for enhanced metal binding capacity </h4>

<p>The assembled plasmids were transformed into TOP10 cells. This meant the mutant libraries were plated on blue/white colony screening plates in TOP10 cells. To transfer the DNA of each white colony on a plate to BL21(DE3) cells would not be feasible, since the MIC of TOP10 cells was not determined (Figure 1) we could not screen for the best mutant. Hence, one mutant was selected at random and transformed to our expression strain.   </p>
<figure>
<img src="https://github.com/idec-teams/2022_Edinburgh-UHAS_Ghana/blob/main/figures/results-2.png?raw=true" style="width: 30vw; left: 0; right: 0; display: block; margin: auto;">
<figcaption><b>Figure 2. a)</b> Illustrates, colony counts of BL21(DE3) recombinantly expressing MTs. In the figure, BL21 (DE3) cells expressing MTs of Danio rerio (DR), Saccharomyces cerevisiae (SC), Mytilus galloprovincialis (MG), and Mytilus edulis (ME) were plated, the wild type is shown in grey and the randomly picked mutant (EP) in red. The figure shows a significant difference in MIC between all the MTs expressing cells and the control. The figure doesn’t indicate if the mutants were better than WTs or vice versa. However, some MTs grew at higher concentration but did not grow at lower concentrations. <b>b)</b> Shows an example of the raw data using WT MT DR plates. The colonies were counted and photographed for each species (<a href="../../supplementary_info/">see Supplementary Information S.3</a>). </figcaption>
</figure>

<p>MTs originating from almost all species, except for S. cerevisiae and M. galloprovincialis grew as lawns at 16 and 17 mg/L of AgNO<sub>3</sub>. The results indicated that there is a sharp decrease in BL21(DE3) growth at 20 mg/L beyond which no lawns were observed for the four species of MTs shown in Figure 3a. Interestingly, in several plated samples (EP MT (DR), WT MT (DR) and WT MT (MG)) no growth was observed in lower concentrations while some colonies grew at higher concentrations. These colonies are good candidates for sequencing analysis and re-plating at higher concentrations. Additionally, the results also indicate that a randomly selected mutant from an EP PCR library did not outperform the wildtype variant of the same species indicating that a screen of the entire mutant library is pertinent.  </p>
<h4 style="text-align: center;">2. Docking simulations </h4>

<p>FASTA sequences of the six MTs were taken from the NCBI database. These sequences were folded, and the Alphafold structures shown in Figure 3 were predicted. These structures were docked to Ag<sup>+</sup> using AutoDock 4.2 such that the structures were hydrated and energy minimised while allowing gamma sulphurs on the sidechains of cysteines to form coordinate covalent bonds with the metal ligand (Figure 3g). The energy minimisation was done after each ligand was docked. MTs contain many cysteines however each cysteine does not carry the same binding affinity for the ligand. This was accounted for using a pass/fail metric (<a href="../../supplementary_info/">see Supplementary Information S.12</a>) where the passed cysteine had negative Gibbs free energy thus making the binding spontaneous.  </p>
<figure>
<img src="https://github.com/idec-teams/2022_Edinburgh-UHAS_Ghana/blob/main/figures/results-3.png?raw=true" style="width: 50vw; left: 0; right: 0; display: block; margin: auto;">
<figcaption><b>Figure 3 a-f) </b>The structures on the left side are generated by AlphaFold using FASTA sequences, and on the right side are the Autodock-generated docking simulations of Ag+ ligands forming coordinate covalent bonds to the WT MT structures. <b>(a)</b> Danio rerio. <b>(b)</b> S. cerevisiae. <b>(c)</b> Mytilus galloprovincialis. <b>(d)</b> Mytilus edulis. <b>(e)</b> Pseudomonas fluorescens. <b>(f)</b> Callinectes sapidus. <b>g)</b> on the left is the undocked structure of D. rerio with all cysteine sidechains visible, the structure on the right is the docked D. rerio structure, which illustrates the conformational change in the gamma sulphurs upon forming coordinate covalent bonds.</figcaption>
</figure>

<p>Furthermore, based on the number of ligands bound per MT a score for the Gibbs free energy of binding per ion was generated for all WT MT sequences (Table 1). The data indicates that <em>P. fluorescens</em> which has the fewest number of cysteines, binds the greatest number of ions, therefore having the lowest total Gibbs free energy and Gibbs free energy of binding per ion (-0.407 kcal/mol). This suggests that <em>P. fluorescens</em> has a high binding efficiency and affinity for Ag<sup>+</sup> ions. Contrastingly, MTs like M. edulis contain more cysteines but bind to fewer Ag<sup>+</sup> ions thus having a lower binding efficiency and affinity.  </p>
<h4 style="text-align:center;">Table 1. In-silico modelled Gibbs free energy</h4>

<table>
    <tr>
        <th>Metallothionein</th>
        <th>Total cysteines</th>
        <th>Number of Ag<sup>+</sup> ions docked</th>
        <th>Total free energy of binding (kcal/mol)</th>
        <th>Gibbs free energy of binding per ion (kcal/mol)</th>
    </tr>
    <tr>
        <td><i>M. edulis </i></td>
        <td>20</td>
        <td>4</td>
        <td>-0.83</td>
        <td>-0.208</td>        
    </tr>
    <tr>
        <td><i>M. galloprovincialis </i></td>
        <td>21</td>
        <td>5</td>
        <td>-0.85</td>
        <td>-1.70</td>        
    </tr>
    <tr>
        <td><i>D. rerio </i></td>
        <td>20</td>
        <td>4</td>
        <td>-0.58</td>
        <td>-0.145</td>        
    </tr>
    <tr>
        <td><i>C. sapidus </i></td>
        <td>18</td>
        <td>5</td>
        <td>-0.65</td>
        <td>-0.130</td>        
    </tr>
    <tr>
        <td><i>P. fluorescens </i></td>
        <td>9</td>
        <td>6</td>
        <td>-2.44</td>
        <td>-0.407</td>        
    </tr>
    <tr>
        <td><i>S. cerevisiae</i></td>
        <td>12</td>
        <td>5</td>
        <td>-1.87</td>
        <td>-0.374</td>        
    </tr>

</table>

<figure>
<img src="https://github.com/idec-teams/2022_Edinburgh-UHAS_Ghana/blob/main/figures/results-4.png?raw=true" style="width: 50vw; left: 0; right: 0; display: block; margin: auto;">
<figcaption><b>Figure 4. a)</b> Multiple sequence alignments of the MTs were followed by conservation analysis against phylogenetically relevant MTs. <b>b)</b> Shows the sequence of <i>Danio rerio</i> metallothionein, with amino acids targeted for potential mutagenesis highlighted in red. </figcaption>
</figure>

<p>MTs contain metal-binding motifs which typically present as: </p>
<p style="text-align: center;">cys-x-cys, cys-x-x-cys, x-cys-cys-x and x-cys-cys-cys-x</p>

<p>Here, x denotes any amino acid other than cysteine (Ziller and Fraissinet-Tachet, 2018). We hypothesised that increasing the number of these motifs in a MT molecule would increase its number of metal-binding sites as long and the protein was functionally folded. </p>
<p>Using Consurf, we analysed our MSA, it generated conservation scores based on phylogenetic analysis. We identified the less conserved regions between two cysteines at least 3 residues apart (Figure 4a). These less conserved sites (Figure 4b) are proposed as possible sites to explore for site-specific mutagenesis using random mutagenic methods like GeneOrator or targeted mutagenesis. We created mutagenic primers for the same (<a href="../../supplementary_info/">see Supplementary Information S.7, S.8</a>). </p>
<p><u><b>1. Discussion</b></u></p>
<p>As our results indicate, we did not screen the entire mutant library (Figure 2a). This was a consequence of several failed DNA assemblies and transformation into BL21 (DE3) cells directly from the JUMP assembly reaction. We observed no colony growth post-transformation, and several colony PCRs showed that the chemically competent BL21 (DE3) cells did not contain the expression plasmid for our MTs. We tried various troubleshooting measures like increasing DNA concentrations in the plasmid assembly reaction, trying different protocols to make a new batch of chemically competent BL21 (DE3) cells. Each assembly was tested via Eco R1 digestion and colony PCR. Given the unsuccessful results, we tried transforming our assemblies onto TOP10 cells (<em>E. coli</em> strain) which was successful, this made it relevant to test the AgNO<sub>3</sub> MIC of TOP10 cells, replacing BL21(DE3) in the screen. It was observed that AgNO<sub>3</sub> was significantly more toxic for TOP10 cells than BL21 (DE3) cells. No growth was observed in TOP10 cells at concentrations as low as 8 mg/L. Consequently, we were not able to establish the MIC of TOP10 cells towards AgNO<sub>3</sub> which meant that we couldn’t successfully establish a starting concentration for a TOP10 cell based mutant screen. We re-designed our experimental pipeline to focus on inferring whether AgNO<sub>3</sub> toxicity was a valid method for a possible BL21(DE3) mutant screen.   </p>
<p>For this, one random mutant MT expressing colony was taken and transformed to BL21(DE3) cells and plated on the AgNO<sub>3</sub> plates, alongside a WT variant of the same species. This result doesn’t indicate if random mutagenesis is the best approach for directed evolution of MTs and other techniques involving site-specific and targeted mutagenesis should be explored. </p>
<p>The results (Figure 2a) show that MT expression does have an impact on the overall MIC of the cell. However, it is difficult to conclude whether the rise in MIC was due to outliers, or metal bioaccumulation. Additionally, this change could be due to cells down-regulating metal transporter proteins, faulty plates with precipitation, or expression of other proteins. To ensure that the original screening hypothesis holds true, the selection needs to be done using controls for all the above stated factors along with the necessary repetition for more accurate data. Protein expression was also not factored into the results, survival at higher concentration for each MT could be because the MTs are chelating more metals per molecule or because the protein expression of 1 MT species is significantly higher than others. </p>
<p>The selection results indicate that the mutants selected did not outcompete WTs for survival, to validate the scope of this screen as a tool to study MTs for bioaccumulation, it would be necessary to do a complete mutant library screen in BL21(DE3) cells. The relevant mutants should be characterised by mass-spec, sequenced, and rescreened along with comparative docking simulations (Figure 1a).  </p>
<p>The docking simulations (Figure 3) and Table 1 was generated using assumptions. The maximum Gibbs free energy (Torsional free energy) was kept constant at 0.60 kcal/mol (<a href="../../supplementary_info/">see Supplementary Information S.13</a>) for all MT species because new atom types cannot be used to calculate the Root Mean Square Deviation (RMSD) of the ligand. This impacts the overall accuracy of the docking.   </p>
<p>The overall experimental output showcased the shift in survival between untransformed BL21 (DE3) cells the MT expressing cells, <em>P fluorescens</em>, showed more “efficient” binding based on the data from Table 1, however, it was never tested experimentally due to flaws in the DNA construct along with <em>C. sapidus</em> which wasn’t tested due to a shortage of resources. </p>
<p>This study shows that there is potential in exploring MTs as an option when considering biotechnological solutions towards problems as scary as the water pollution crisis in Ghana. The novel screening method explored here has not been characterised fully but leaves room for further directed evolution experiments and a platform for doing so has also been provided with the proposal for alternate methods of random site-specific mutagenesis. </p>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
    
    <nav class="md-footer__inner md-grid" aria-label="Footer" >
      
        
        <a href="../design/" class="md-footer__link md-footer__link--prev" aria-label="Previous: Design" rel="prev">
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
          </div>
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Previous
              </span>
              Design
            </div>
          </div>
        </a>
      
      
        
        <a href="../../report/" class="md-footer__link md-footer__link--next" aria-label="Next: &lt;b&gt;Report&lt;/b&gt;" rel="next">
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Next
              </span>
              <b>Report</b>
            </div>
          </div>
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4Z"/></svg>
          </div>
        </a>
      
    </nav>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Copyright &copy; 2022 iDEC. CC-BY-4.0 | The International Directed Evolution Competition (iDEC) is a Scottish Charitable Incorporate Organisation (SCIO), registered in Scotland, with charity number：SC050826
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.sections"], "search": "../../assets/javascripts/workers/search.5bf1dace.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../../assets/javascripts/bundle.078830c0.min.js"></script>
      
        <script src="../../javascripts/config.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.2.1/tablesort.min.js"></script>
      
        <script src="../../javascripts/tables.js"></script>
      
    
  </body>
</html>